docs/monitoring/monitoring.md

```diff
@@ -16,3 +16,59 @@ Also, users can click the [Grafana](http://grafana.com/) dashboard link to view
 ## VM Detail Metrics
 For each VM, users can view the VM metrics by clicking the VM details page.
 ![](./assets/vm-metrics.png)
+
+
+## Change resources setting of Monitoring
+_Available as of v1.0.1_
+
+Monitoring has a couple of components to collect metrics data from all NODEs/PODs/VMs/... and summarize them. The resources required by Monitoring are related to workloads and hardware resources. Harvester sets the default values according to engineering practices, users can change them accordingly.
+
+The resources setting of two of the components are available
+
+(1) Monitoring-Prometheus
+
+(2) Monitoring-Prometheus-node-exporter
+
+### Change resources setting of Monitoring-Prometheus
+
+In `Advanced` `Settings` page, user can view and change the resources setting of Monitoring.
+
+(1) Naviate to settings page, find `harvester monitoring`
+![](./assets/monitoring-setting.png)
+
+(2) Click `Show harvester-monitoring` to view current value of setting
+![](./assets/monitoring-setting-view-current.png)
+
+(3) Click up-right corner pop up menu and select `Edit Setting` to set new value
+![](./assets/monitoring-setting-edit-config.png)
+
+(4) Click `Save`, wait a while, the monitoring will be restarted with new resources setting
+
+The most frequently used option is memory setting.
+
+`Requested Memory` is the minimum memory of the `Monitoring`, the recommended value is about 5% to 10% of system memory of one single management node. A value of less than 500Mi will be denied. 
+
+`Memory Limit` is the maximum memory of the `Monitoring`, the recommended value is about 30% of system memory of one single management node, when monitoring touches this value, it will be restarted.
```

Comment:
```suggestion
`Memory Limit` is the maximum memory of the `Monitoring`. The recommended value is about 30% of system memory of one single management node, when the `Monitoring` reaches this value, it will be restarted.
```